packages/manager/modules/vps/src/dashboard/migrate/migrate-confirm/migrate-confirm.html

```diff
@@ -1,7 +1,7 @@
 <form novalidate name="form">
     <oui-modal
         data-primary-label="{{ ::'vps_dashboard_migrate_confirm_button_text' | translate }}"
-        data-primary-action="$ctrl.migrateVps()"
+        data-primary-action="$ctrl.migrateVps($ctrl.selectedPlan)"
```

Comment:
Don't necessary to pass the parameter.
In `migrateVps()` function you don't take in account parameter.

```suggestion
        data-primary-action="$ctrl.migrateVps()"
```

Reply:
sure Zahaf I will remove it.